src/pages/location-message/location-message.html

```diff
@@ -0,0 +1,22 @@
+<ion-header>
```

Comment:
It's about the file not the line, I meant that you should define more classes

```
the rationale behind this is that it would be easier to look at, and the view would be more organized. If you're convinced that it's unnecessary, I'm with you.
```